server/chat-commands.js

```diff
@@ -3740,6 +3740,39 @@ const commands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ & ~`],
 
+	acceptdraw: 'acceptdraw',
+	offerdraw(target, room, user, connection, cmd) {
+		const battle = room.battle;
+		const playerIDS = Object.keys(battle.playerTable);
+		if (!battle) return this.errorReply("Must be in a battle room.");
+		if (!Config.allowrequestingties) return this.errorReply("This server does not allow the offering/accpeting of draws");
+		if (!this.can('roomvoice', null, room)) return;
+		if (cmd === 'acceptdraw' && !battle.allowTie) return this.errorReply("No draw offer available. It might have been automatically withdrawn as the current turn started.");
+		if (!battle.allowTie) {
+			battle.allowTie = [];
+			for (const player of playerIDS) {
+				if (player === user.userid) {
+					battle.allowTie.push(player);
+					continue;
+				}
+				Users(player).sendTo(
+					room,
+					Chat.html`|html|${user.name} is offering a draw, <button class="button">do you accept</button>`
```

Comment:
Not sure about the wording here, but probably only `'accept'` should be in the `button`

Reply:
Sure, I also forgot to make the button send the command, lol.